Library/Homebrew/cask/cmd/audit.rb

```diff
@@ -44,7 +45,7 @@ def run
         end
         casks = casks.map { |c| CaskLoader.load(c, config: Config.from_args(args)) }
         any_named_args = casks.any?
-        casks = Cask.to_a if casks.empty?
+        casks = Cask.all if casks.empty?
```

Comment:
This deserves scrutiny: `Cask.to_a` isn't found by sorbet, I made this change based on [this](https://github.com/Homebrew/brew/commit/3a709d36b28de8510e537431b8256b588d934e5c#diff-2c22a8e4df519ede038d14ceab4b608b37dd474043557c7eea9b79dc0c7a4f9cL107) refactor

Reply:
```rb
~ $ brew irb
==> Interactive Homebrew Shell
Example commands available with: `brew irb --examples`
irb(main):001:0> Cask.respond_to? :to_a
=> false
irb(main):002:0> Cask.to_a
Traceback (most recent call last):
        2: from /usr/local/Homebrew/Library/Homebrew/brew.rb:93:in `<main>'
        1: from (irb):2
NoMethodError (undefined method `to_a' for Cask:Module)
Did you mean?  to_s
```

It makes me wonder if this is dead code though and can just be removed.

Reply:
All of these files should be removed instead: https://github.com/Homebrew/brew/issues/14799